### ISSUE TITLE
Getting 401 when using Github branches

### ISSUE THREAD

**Issue by kjartanm** [mavoweb/mavo#550]
Had problems when using branch as storage. Solved it for myself by adding data-param in request() call, using following code in backend.github.js, line 49:

```
return this.request(info.apiCall, {ref:this.branch}, "GET", {
        headers: {
            "Accept": "application/vnd.github.squirrel-girl-preview"
        }
    }).then(response => Promise.resolve(info.repo? _.atob(response.content) : response));
```

**Comment by LeaVerou** [comment]
Hi @kjartanm, would you consider submitting a PR with your change?

**Comment by kjartanm** [comment]
Hi @LeaVerou ,
Would love to, but do I need some sort of access to the repo before submitting PR? I've tried to submit one, but haven't succeeded.

**Comment by LeaVerou** [comment]
Nope! That's the whole point of pull requests, that you don't need any access to submit one!

**Comment by LeaVerou** [comment]
Fixed via #561
